### ISSUE TITLE
[BUG] The new Balck cleaver passive Rage (11.5) is not granting the expected number of movement speed vs 2+ enemies.

### ISSUE THREAD

**Issue by Alex-Tsvetanov** [RiotGames/developer-relations#461]
**Bug Description**  
Here's the thing. The new passive of Black cleaver (11.5 changes) grants you 5 movement speed per stacks of Carve (the armor reduction passive) but if you have stacks on more than 1 enemy the number of stacks is not summed and you are actually getting less movement speed than it's expected. 

In [this video](https://youtu.be/KW695KnO-Ec), I hit 2 dummies with Aatrox's Q1 and I got 5 movement speed instead of 2x5=10 movement speed. 

So to fix it, Riot Games should: 
Option 1: change the Rage passive description to "Dealing physical damage to a champion grants you 5 bonus movement speed per stack of Carve on the enemy with the most stacks of Carved for 2 seconds, up to 30."
Option 2: change the Carved passive description (and behavior) to "Dealing physical damage to an enemy champion grants you 4% armor penetration for 6 seconds, stacking up to 6 times for a maximum of 24% armor penetration."
Option 3: change the Rage passive so it grants 5 MS per stack of Carved no matter if those stacks are on multiple enemies or single enemy champion(s). 

The video with the explanation and in-game prove is [here](https://youtu.be/KW695KnO-Ec).

**Problem Description**  
The Black Cleaver Rage passive is "not working as described"/"not described as it works".

**Expected Result**  
Rage passive should provide 5 MS per stack no matter the number of enemies with stacks of Carved or the most stacks on a single enemy.

**Actual Result**  
The passive provides 5 MS per stack of Carved based on the highest number of stacks on a given enemy.

**Developer Impact**  
Nothing aside it confuses the player base.

* Frequency: Every game.
* Severity: When it happens, the whole player base is confused.
* Cost: Nothing except it destroys my analysis over the meta changes
* Efficiency: Requires me to test every interaction in-game so I can make sure I run my analysis correctly

**Player Impact**  
The issue confuses the players making them wondering where is their movement speed.

* Frequency: Every time the user builds Black clever.
* Severity: When this bug happens, a user is unable to stick to the enemies which he is supposed to reach out.
* Quantity: As many times as you damage more than a single enemy.
* Region: This affects for sure players in EUNE, but I think it is in every server.

**Preconditions**  
1) You should have built Black clever.
2) You should have different numbers of Carved stacks on different enemies.

**Repro Steps**  
This is a replication of the issue with voice commentary [YouTube Video of the issue](https://youtu.be/KW695KnO-Ec)

**Comment by Hi-Ray** [comment]
Hey there!

**TLDR: No help/support at all will be given and this issue will be closed**

This isn't the page to ask for that for help with things like that
please contact support through support@riotgames.com or go to

https://support.riotgames.com/hc/en-us

We ask that you close this issue as this is where issues regarding
the Riot Games API (A tool which provides data to sites like op.gg)
are posted.

Posting with issues regarding support or gameplay issues will simply
be ignored and most likely be closed anyway, Lorem ipsum dolor sit amet, 
consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore 
et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
 ullamco laboris nisi ut aliquip ex ea commodo consequat. 
It is advised you contact support with the details provided above.

Kind Regards

**Comment by Alex-Tsvetanov** [comment]
Hello!

Excuse me. The Discord server of Riot Games Third-Party Devs told me that it is an option to put it here. I am sorry for the misunderstanding and I will close it immediately.

Regards
